client/py/aou_workbench_client/config.py

```diff
@@ -4,11 +4,13 @@
 import json
 
 # Constants for keys in the configuration
-ALL_OF_US_CONFIG_FILE = 'all_of_us_config.json'
+ALL_OF_US_CONFIG_FILE = '~/config/all_of_us_config.json'
 WORKSPACE_NAMESPACE_KEY = 'WORKSPACE_NAMESPACE'
 WORKSPACE_ID_KEY = 'WORKSPACE_ID'
 API_HOST_KEY = 'API_HOST'
 BUCKET_NAME_KEY = 'BUCKET_NAME'
+CDR_VERSION_PROJECT = 'CDR_VERSION_PROJECT'
```

Comment:
opt_nit: Consider naming these more specifically. Project/dataset are fairly overloaded even within the PMI context. Maybe `CDR_VERSION_CLOUD_PROJECT` and `CDR_VERSION_BIGQUERY_DATASET`.

Reply:
Done.